Dockerfile

```diff
@@ -21,19 +21,28 @@ FROM ubuntu:22.04 AS builder
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y \
         cargo \
-        clang-15 \
         cmake \
         g++ \
         git \
         libz3-dev \
-        llvm-15-dev \
-        llvm-15-tools \
         ninja-build \
         python3-pip \
         zlib1g-dev \
+        wget \
+        lsb-release \
+        software-properties-common \
+        gnupg \
+        libzstd-dev \
     && rm -rf /var/lib/apt/lists/*
 RUN pip3 install lit
 
+# Install LLVM
+RUN mkdir /llvm
+WORKDIR /llvm
+RUN wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 15
```

Comment:
What's the motivation for switching to packages from the LLVM project? Don't we use the Dockerfile also to check compatibility with the Ubuntu's LLVM packages?

Reply:
This script is very convenient to get any version of llvm, which is not always very easy with ubuntu, especially for the latest versions.
I didn't know the goal was also to test ubuntu package, let's keep taking it from ubuntu then.